EpicLoot/config/magiceffects.json

```diff
@@ -1581,11 +1610,86 @@
         "$mod_epicloot_me_modifyjumpstaminause_suffix1"
       ]
     },
+    {
+      "Type" : "ModifyAttackEitrUse",
+      "DisplayText" : "$mod_epicloot_me_modifyattackeitruse_display",
+      "Description" : "$mod_epicloot_me_modifyattackeitruse_desc",
+      "Requirements" : {
+        "AllowedItemTypes" : [
```

Comment:
For all of these additions you only need to define AllowedItemTypes or ExcludedItemTypes, not both. I would choose whichever is simpler for the effect keeping in mind how it may affect modded items that use unconventional item type assignments.

Reply:
Fixed in: https://github.com/OrianaVenture/Randy_Vapok_ValheimMods/pull/10/commits/172dbd3606de158db2014d31ece196befcce7020